src/lib/core/option/_option-theme.scss

```diff
@@ -48,7 +48,7 @@
 @mixin mat-option-typography($config) {
   .mat-option {
     font: {
-      family: mat-font-family($config, subheading-2);
+      font-family: mat-font-family($config);
```

Comment:
This will produce `font-font-family: <value>` when transpiled.

Reply:
Whoops, thanks.